components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -878,11 +878,6 @@ func NewKymaOrchestrationProcessingQueue(ctx context.Context, db storage.BrokerS
 		step     upgrade_kyma.Step
 		cnd      upgrade_kyma.StepCondition
 	}{
-		{
-			weight: 2,
```

Comment:
note: this was not supposed to be moved. Explanation in https://github.com/kyma-project/control-plane/pull/1258.